pyvrp/cpp/search/LocalSearch.cpp

```diff
@@ -229,16 +231,15 @@ void LocalSearch::applyEmptyRouteMoves(Route::Node *U,
 {
     assert(U->route());
 
-    auto begin = routes.begin();
-    for (size_t vehType = 0; vehType != data.numVehicleTypes(); vehType++)
+    for (auto const &[vehType, offset] : orderVehTypes)
```

Comment:
I currently understand why we do this randomly/in the order of a shuffled list. A year or so from now I definitely will no longer remember that. Can we add a comment here along the lines of:
```cpp
    // We apply moves involving empty vehicles (routes) in random order (given
    // by orderVehTypes). This helps because empty vehicle moves incur fixed
    // cost, and a purely greedy algorithm over-prioritises vehicles with low
    // fixed costs (but possibly high variable costs).
```